dbt/adapters/athena/column.py

```diff
@@ -0,0 +1,20 @@
+from dbt.adapters.base import Column as BaseColumn
+
+
+class AthenaColumn(BaseColumn):
+    @property
+    def data_type(self) -> str:
+        # NOTE: Athena has different types between DML and DDL, use DDL type in this case
+        #       ref: https://docs.aws.amazon.com/athena/latest/ug/data-types.html
+        if self.dtype.lower() == "integer":
+            return "int"
+        elif self.is_string():
+            return self.string_type(self.string_size())
```

Comment:
Doesn't athena pretend to have always strings?

Reply:
Athena supports `string` and `varchar(n)`, so I choose this implementation, but I think `string` is clearer than `varchar(n)`, so I'll replace this whit `string`

<img width="375" alt="image" src="https://user-images.githubusercontent.com/16137578/202910894-7e0fa266-276f-4905-9dc2-eb2b179ef9c1.png">

Reply:
as we would like to use the same approach for iceberg, and there varchar is not supported, also IMHO we should stick with string.